wiki/nomad-books.md

```diff
@@ -15,7 +15,7 @@ whenever and wherever they wish.
 
 A nomad book is an item that can store a 3x1x3 sized camp. The book displays a campfire as indicator if the camp is deployed (campfire absent) or not (campfire visible).
 
-![Campfire Indicator](nomad-books/CampfireIndicator.png){: .wiki}
```

Comment:
Doesn't this change how the image displays ?

Reply:
Didn't you say you removed the whole {: .wiki} stuff?

Reply:
I forgor 💀 
There is still specific styling for those images, but I don't know if it's actually relevant. Does it make any visible difference in your testing ?

Reply:
Let me update the test website rq

Reply:
Nope, not in testing at least

Reply:
From my local testing it does change the spacing around images :
<img width="1108" alt="image" src="https://github.com/Ladysnake/Ladysnake.github.io/assets/25536265/6f95032c-8131-42c8-9884-8877e3dbd484">
vs
<img width="1108" alt="image" src="https://github.com/Ladysnake/Ladysnake.github.io/assets/25536265/04f5c2ab-34f1-4bfe-9f3a-206acb97d46b">

Reply:
Ah true, didn't spot that, at least with the first image it looks better tho xD